lib/value/src/kind/builder.rs

```diff
@@ -35,30 +36,12 @@ impl Kind {
             timestamp: None,
             regex: None,
             null: Some(()),
+            undefined: Some(()),
             array: Some(Collection::json()),
             object: Some(Collection::json()),
         }
     }
 
-    /// The "primitive" type state.
```

Comment:
Was this not used before? I didn't notice an equivalent replacement.

Reply:
This was not used. Originally many similar methods were added to `Kind` that weren't used yet, assuming they would be useful. We are pretty far into using `Kind` now and we haven't seen a use for this yet, so I figured it was time to remove it.
I was originally trying to decide if `undefined` would be considered a "primitive" or not, and didn't see an example usage.